src/sentry/backup/exports.py

```diff
@@ -146,45 +116,34 @@ def filter_objects(queryset_iterator):
     def yield_objects():
         # Collate the objects to be serialized.
         for model in sorted_dependencies():
-            # This is a bit confusing, but what it's saying is: any model that does not have
-            # `__relocation_scope__` set MUST be a non-Sentry model (we check this both in init-time
-            # testing and at runtime startup). We "deduce" a `RelocationScope` setting for this
-            # non-Sentry model based on the config the user passed in: if they set
-            # `old_config.include_non_sentry_models` to `True`, we set all deduced non-Sentry
-            # relocation scopes to `Global`. Otherwise, we just exclude them.
-            # print(f"Deduced rel scope for {model.__name__}: {inferred_relocation_scope.name}\n")
-            includable = old_config.include_non_sentry_models
-            if hasattr(model, "__relocation_scope__"):
-                # TODO(getsentry/team-ospo#186): This won't be sufficient once we're trying to get
-                # relocation scopes that may vary on a per-instance, rather than
-                # per-model-definition, basis. We'll probably need to make use of something like
-                # Django annotations to efficiently filter down models.
-                if getattr(model, "__relocation_scope__") in allowed_relocation_scopes:
-                    includable = True
-
-            if (
-                not includable
-                or model.__name__.lower() in old_config.excluded_models
-                or model._meta.proxy
-            ):
-                printer(f">> Skipping model <{model.__name__}>", err=True)
+            includable = False
+
+            # TODO(getsentry/team-ospo#186): This won't be sufficient once we're trying to get
+            # relocation scopes that may vary on a per-instance, rather than
+            # per-model-definition, basis. We'll probably need to make use of something like
+            # Django annotations to efficiently filter down models.
+            if getattr(model, "__relocation_scope__", None) in allowed_relocation_scopes:
+                includable = True
+
+            if not includable or model._meta.proxy:
```

Comment:
What was your reasoning behind removing the `>> Skipping model` ?

Reply:
I just felt it was a bit redundant, and kind of clogged up the screen. If anything, we'd probably want to tell people the opposite: which models we exported, rather than the ones we skipped, since the latter setup assumes they know the full set of models.

In any case, we're moving away from the "just export the entire table blindly" backup setup we had in the past. Rather than just exporting all models of a certain kind or not, we now have more nuanced logic. For example, we may export most instances of model X, but leave out a few because the users or orgs that own them have been filtered out from the export. So I think these sorts of comments are too coarse-grained for how exports work now.

I do think that some feedback from the export script is nice though - I'll think about a good way to add that in a future PR.